### DISCUSSION TITLE
Does hasura support redis-cluster?

### DISCUSSION THREAD

**Discussion by miran248** [hasura/graphql-engine#10611] [Question] [answered]
Hey,

When i deploy redis-cluster (3+3), hasura only uses one instance, it seems (all instances, except 1+1, are empty).
I'm only setting the following env var
`HASURA_GRAPHQL_REDIS_URL: redis://name-redis-cluster-headless.namespace.svc.cluster.local:6379`
Do i need to set any other env var?
Maybe pass individual (comma separated) urls to the above env var?

Not sure if i'm looking at the right place, but hedis supports clustered connections https://github.com/hasura/graphql-engine/blob/v2.44.0/server/forks/hedis/src/Database/Redis/Connection.hs#L194

Versions:
hasura `hasura/graphql-engine:v2.44.0` ([custom setup](https://github.com/248-sh/kubernetes-manifests/blob/v1.6.7/hasura/base/deployment.yaml))
redis-cluster `bitnami/redis-cluster:7.4.1-debian-12-r2` ([helm chart](https://github.com/bitnami/charts/tree/redis-cluster/11.0.8/bitnami/redis-cluster))

Thanks!
Miran

**Comment by robertjdominguez** [comment]
@miran248 sorry for the delay here!

@hgiasac — would you be able to weigh in on the best way to resolve this redis issue with v2?

**Comment by miran248** [comment]
I have since switched to the regular redis with sentinels and that worked, but it would still be helpful if it was documented somewhere, whether it's supported.

Thanks for the reply!

**Comment by robertjdominguez** [comment]
Good to know, @miran248! I'll wait for @hgiasac to drop a reply and we can work to update the docs 👍

**Comment by hgiasac** [comment, marked as answer]
@miran248 @robertjdominguez Yes, Redis cluster has been supported since [v2.28](https://github.com/hasura/graphql-engine/releases/tag/v2.28.0)
